apps/robots-generator/.eslintrc.json

```diff
@@ -0,0 +1,33 @@
+{
+  "root": true,
+  "extends": ["next/core-web-vitals"],
+  "rules": {
+    "import/order": [
+      "error",
+      {
+        "alphabetize": {
+          "order": "asc",
+          "caseInsensitive": true
+        },
+        "newlines-between": "always"
+      }
+    ],
+    // https://mui.com/guides/minimizing-bundle-size/#option-1
```

Comment:
Is this being used on other apps as well? Not 100% sure but I believe we're using option 2 with [`modularizeImports` / `optimizePackageImports`](https://nextjs.org/docs/app/api-reference/next-config-js/optimizePackageImports).